### PR TITLE
[bigquery] Increase Storage Write API max request size

### PR DESCRIPTION
**Pull request by nathan-artie**
Also kill `batchSize` config.

### CODE REVIEW

File: clients/bigquery/bigquery.go [hunk 1]
```diff
@@ -34,13 +34,12 @@ const (
 	describeNameCol               = "column_name"
 	describeTypeCol               = "data_type"
 	describeCommentCol            = "description"
-	// Storage Write API is limited to 10 MB, let's start out conservative and use 80% of that.
-	maxRequestByteSize = 10_000_000 * .8
+	// Storage Write API is limited to 10 MiB, subtract 50 KiB to account for request overhead.
+	maxRequestByteSize = (10 * 1024 * 1024) - (50 * 1024)
```
Comment (Tang8330) [inline clients/bigquery/bigquery.go:38]: is 50 kb fixed?

File: clients/bigquery/bigquery.go [hunk 2]
```diff
@@ -223,7 +222,6 @@ func (s *Store) Dedupe(tableID sql.TableIdentifier, primaryKeys []string, includ
 }
 
 func LoadBigQuery(cfg config.Config, _store *db.Store) (*Store, error) {
-	cfg.BigQuery.LoadDefaultValues()
```
Comment (Tang8330) [inline clients/bigquery/bigquery.go:226]: Nice

**Review by Tang8330 — APPROVED** [review]

File: clients/bigquery/bigquery.go (hunk 1 above)
Comment (nathan-artie) [reply clients/bigquery/bigquery.go:38]: After a bunch of testing the cutoff seemed to be around 30 KiB so I decided to go with 50 KiB to be safe.

**Comment by Tang8330** [comment]
i c

File: clients/bigquery/bigquery.go (hunk 1 above)
Comment (nathan-artie) [reply clients/bigquery/bigquery.go:38]: I tested it it with a pretty wide table so I think it should be fine, but if we do see errors we can increase it to 100 KiB.

**Pull request merged**